tests/xtro-sharpie/iOS-MapKit.ignore

```diff
@@ -8,7 +8,4 @@
 
 !missing-field! MKAnnotationCalloutInfoDidChangeNotification not bound
 !missing-protocol-conformance! MKUserLocation should conform to MKAnnotation
-!missing-protocol-member! MKOverlay::coordinate not found
-
-## Getting "[FAIL] Selector not found for MapKit.MKMapItem : encodeWithCoder:" when 'MKMapItem' implements 'NSSecureCoding' on 32-bit iOS
-!missing-protocol-conformance! MKMapItem should conform to NSSecureCoding (defined in 'MKMapItemSerialization' category)
\ No newline at end of file
+!missing-protocol-member! MKOverlay::coordinate not found
```

Comment:
For the future: make sure to end files with a newline to avoid diff noise :smile:

Reply:
And I should stop commenting until I've read all the commits...